jobs/build/oc_sync/publish-clients-from-payload.sh

```diff
@@ -24,44 +34,99 @@ else
+    pushd ${OUTDIR}
+    # External consumers want a link they can rely on.. e.g. .../latest/openshift-client-linux.tgz .
+    # So whatever we extract, remove the version specific info and make a symlink with that name.
+    for f in *.tar.gz *.bz *.zip *.tgz ; do
 
+        # Is this already a link?
+        if [[ -L "$f" ]]; then
+            continue
+        fi
 
-# External consumers want a link they can rely on.. e.g. .../latest/openshift-client-linux.tgz .
-# So whatever we extract, remove the version specific info and make a symlink with that name.
-for f in *.tar.gz *.bz *.zip *.tgz ; do
+        # example file names:
+        #  - openshift-client-linux-4.3.0-0.nightly-2019-12-06-161135.tar.gz
+        #  - openshift-client-mac-4.3.0-0.nightly-2019-12-06-161135.tar.gz
+        #  - openshift-install-mac-4.3.0-0.nightly-2019-12-06-161135.tar.gz
+        #  - openshift-client-linux-4.1.9.tar.gz
+        #  - openshift-install-mac-4.3.0-0.nightly-s390x-2020-01-06-081137.tar.gz
+        #  ...
+        # So, match, and store in a group, any non-digit up to the point we find -DIGIT. Ignore everything else
+        # until we match (and store in a group) one of the valid file extensions.
+        if [[ "$f" =~ ^([^0-9]+)-[0-9].*(tar.gz|tgz|bz|zip)$ ]]; then
+            # Create a symlink like openshift-client-linux.tgz => openshift-client-linux-4.3.0-0.nightly-2019-12-06-161135.tar.gz
+            ln -sfn "$f" "${BASH_REMATCH[1]}.${BASH_REMATCH[2]}"
+        fi
+    done
+    popd
+}
 
-    # Is this already a link?
-    if [[ -L "$f" ]]; then
-        continue
+function extract_opm() {
+    OUTDIR=$1
+    mkdir -p "${OUTDIR}"
+    OPERATOR_REGISTRY=$(oc adm release info --image-for operator-registry "$PULL_SPEC")
+    # extract opm binaries
+    BINARIES=(opm)
+    PLATFORMS=(linux)
+    if [ "$ARCH" == "x86_64" ]; then  # For x86_64, we have binaries for macOS and Windows
+        BINARIES+=(darwin-amd64-opm windows-amd64-opm)
+        PLATFORMS+=(mac windows)
     fi
 
-    # example file names:
-    #  - openshift-client-linux-4.3.0-0.nightly-2019-12-06-161135.tar.gz
-    #  - openshift-client-mac-4.3.0-0.nightly-2019-12-06-161135.tar.gz
-    #  - openshift-install-mac-4.3.0-0.nightly-2019-12-06-161135.tar.gz
-    #  - openshift-client-linux-4.1.9.tar.gz
-    #  - openshift-install-mac-4.3.0-0.nightly-s390x-2020-01-06-081137.tar.gz
-    #  ...
-    # So, match, and store in a group, any non-digit up to the point we find -DIGIT. Ignore everything else
-    # until we match (and store in a group) one of the valid file extensions.
-    if [[ "$f" =~ ^([^0-9]+)-[0-9].*(tar.gz|tgz|bz|zip)$ ]]; then
-        # Create a symlink like openshift-client-linux.tgz => openshift-client-linux-4.3.0-0.nightly-2019-12-06-161135.tar.gz
-        ln -sfn "$f" "${BASH_REMATCH[1]}.${BASH_REMATCH[2]}"
+    MAJOR=$(echo "$VERSION" | cut -d . -f 1)
+    MINOR=$(echo "$VERSION" | cut -d . -f 2)
+    if [ "$MAJOR" -eq 4 ] && [ "$MINOR" -le 6 ]; then
```

Comment:
I do not believe we want to publish this for releases before 4.6. can you please exclude them?

Reply:
ack. will update it in the next push.